downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -1,46 +1,218 @@
+
+``` ballerina
+configurable map<string> admin = ?;
+
+type HttpResponse record {|
+    string method;
+    string httpVersion = "HTTP_1_0";
+    map<string> headers;
+|};
+
+configurable HttpResponse response = ?;
+configurable map<string>[] users = ?;
+```
+
+the values can be provided in the `Config.toml` as follows.
+
+```toml
+[admin]
+username = "John Doe"
+mail = "John@hotmail.com"
+location = "LK"
+
+[response]
+method = "POST"
+httpVersion = "HTTP_2_0"
+headers.Status = "200 OK"
+headers.Content-Type = "text/html"
+
+[[users]]
+username = "Jack"
+location = "UK"
+
+[[users]]
+username = "Jane"
+location = "US"
+```
+
+### Breaking Changes
+
+- The `io.ballerina.runtime.api.types.Type#getName` and `io.ballerina.runtime.api.types.Type#getQualifiedName` methods now return an empty string if no name was associated with the type. The `io.ballerina.runtime.api.types.Type#toString` method can be used to get the string representation of a type.
+- `Wait` actions that wait for expressions that are not named workers can return errors now. The eventual type of such wait future expressions is now `T|error` where `T` is the type of the original return value.
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `graphql` Package
+- Added declarative auth configurations
+- Added support for optional types as inputs
+- Added support for returning unions of distinct service objects
+- Added support for inline fragments
+- Added support for enums as input values
+
+### Improvements
+
+#### `cache` Package
+Marked the `cache:Cache` class as an `isolated` class.
+
+#### `file` Package
+Marked the `file:Listener` class as an `isolated` class.
+
+#### `graphql` Package
+- Improved introspection validation and execution
+- Added missing fields in the GraphQL types
+- Improved the compiler plugin to validate inputs and return types
+- Added support to use included record parameters instead of the `record` type in the listener initialization
+
+#### `http` Package
+- Improved the `http:Client` remote methods to support inferring the target type from the contextually-expected type
+- Changed the listeners and clients to use include record parameters as configuration parameters
+- Populated the error detail of `http:ClientRequestError` and `http:RemoteServerError` with the response payload and the headers
+- Swapped the `mediaType` and `headers` arguments in the client remote methods
+
+#### `sql` Package
+- Added the SQL Array Value type support and introduced the new distinct array value types for identified SQL types
+- Marked the `sql:Client` class as an `isolated` class
+
+#### `websubhub` Package
+Included the auth configuration in the WebSubHub publisher client configuration.
```

Comment:
```suggestion
- Included the auth configuration in the WebSubHub publisher client configuration.
```

Reply:
We don't need bullet points if there is no list and only one point.